win32ss/gdi/ntgdi/freetype.c

```diff
@@ -6020,15 +6153,15 @@ GreExtTextOutW(
         {
             FT_Vector delta;
             FT_Get_Kerning(face, previous, glyph_index, 0, &delta);
-            TextLeft += delta.x;
+            TextLeft64 += delta.x;
         }
-        DPRINT("TextLeft: %I64d\n", TextLeft);
-        DPRINT("TextTop: %lu\n", TextTop);
-        DPRINT("Advance: %d\n", realglyph->root.advance.x);
+        //DPRINT("TextLeft64: %I64d\n", TextLeft64);
+        //DPRINT("TextTop64: %I64d\n", TextTop64);
+        //DPRINT("Advance: %d\n", realglyph->root.advance.x);
```

Comment:
May be re-enabled again? (It's DPRINT so it should not be display in principle, unless you disable the `#define NDEBUG` at the top of the source file.)